### PR TITLE
Grid + Nomeclature change

### PR DESCRIPTION
**Pull request by gamaanderson**
I did something that may be I shouldn't: to match the updates in grid display I changes the nomenclature of other parts of the code.

* changed `Display` to `RadarDisplay` and `GridDisplay` 
* changed **tilt.py** by **level.py** as well as the names inside, and add a variable name Vlevel that may refer to Vtilt, VlevelZ, VlevelY or VlevelX
* in `RadarDisplay` changed scan_type to plot_type with names "radarPpi", "radarRhi" or "radarAirborne" to match "gridZ", "gridY" and "gridX" in  `GridDisplay` 
* add variable name Vcontainer that may refer to Vradar or Vgrid

### CODE REVIEW

File: artview/components/level.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
+"""
+level.py
+
+Class instance used for modifying level via Display window.
+"""
```
Comment (nguy) [inline artview/components/level.py]: My only real comment on the nomenclature changes is that we may want to add wording in some scripts that notes that levels = tilts for "radar" files.

Comment (gamaanderson) [inline artview/components/level.py]: it is not much, but we have this [wording](https://github.com/gamaanderson/artview/blob/grid/artview/core/core.py#L56-L58)

Comment (gamaanderson) [inline artview/components/level.py]: There is also wording [here](https://github.com/gamaanderson/artview/blob/grid/artview/components/level.py#L40)

File: artview/components/plot_radar.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def __init__(self, Vradar, Vfield, Vtilt, Vlims=None, Vcmap=None,
         # Connect the components
         self.connectAllVariables()
 
-        self.scan_type = None
+        self.plot_type = None
 
         # Set plot title and colorbar units to defaults
         self.title = None
```
Comment (nguy) [inline artview/components/plot_radar.py]: L149 still has a reference to ```Display```.

File: artview/components/level.py (hunk 1 above)
Comment (nguy) [inline artview/components/level.py]: Okay that second was what I was looking for.

**Comment by nguy** [comment]
I had not tested the option such as ```python -m artview -F filename```. This returns an empty directory and fails to open on my machine.

**Comment by nguy** [comment]
@gamaanderson If you resubmit, can you also replace:

standard.py
Line 8 ```def run(DirIn='./', filename=None, field=None):```

with ```def run(DirIn=os.getcwd(), filename=None, field=None):```

an import of os will also be required.

**Comment by gamaanderson** [comment]
done! other update from my working branch came with it, but it is just grid related so I don't think it is a problem to merge it

**Comment by nguy** [comment]
Awesome! Thanks for that.

**Pull request merged**